integration_tests/test_list_03.py

```diff
@@ -10,7 +10,14 @@ def test_list(n: i32) -> i32:
         sum += a[i]
     return sum
 
+def test_list_02(pos: i32, value: i32) -> i32:
+    x: list[i32]
+    x = [1, 3, 4]
+    x.insert(pos, value)
```

Comment:
Let's test multiple inserts on the same list variable.

Reply:
Now, I understood what the issue is! Instead of inserting at the `pos`, I replace it.
Thank you

Reply:
Yes. Writing moderate to strong tests is important to implement the feature robustly. The above was very simple. Leaving out corner cases is okay but leaving out a large class of use cases is not recommended.